templates/logrotate_node_exporter.j2

```diff
@@ -0,0 +1,9 @@
+/var/log/node_exporter/node_exporter.log {
+    daily
+    rotate {{ node_exporter_log_days_keep }}
+    compress
+    shred
+    postrotate
+      /usr/bin/killall -HUP rsyslogd
```

Comment:
Can we use pkill as a better habit. 

Killall on some (non Linux) systems does actually just kill alll from PID 1 downwards

Reply:
I've looked into pkill vs killall a bit, and it seems that killall is actually a bit safer on linux systems than pkill, because it takes a complete process name and pkill takes a pattern. I guess I should probably still use pkill.

Reply:
```
-x, --exact
              Only match processes whose names (or command line if -f is specified) exactly match the pattern.
```
It's not a hard requirement, but it's just something i personally try to avoid doing for that one time that you accidentally end up on one of them.

Reply:
/me should read man pages before complaining ._.